src/scripts.js

```diff
@@ -245,6 +262,43 @@ function makeFriendChallengeHTML(id, activityInfo, userStorage, method) {
 function makeStepStreakHTML(id, activityInfo, userStorage, method) {
   return method.map(streakData => `<li class="historical-list-listItem">${streakData}!</li>`).join('');
 }
-
 // startApp();
+const addClass = (element, className) => {
+  element.classList.add(className || "hidden");
```

Comment:
Good idea on these add and remove functions. Maybe we could go further and make them take in an array?